apps/web-giddh/src/app/vat-report/obligations/obligations.component.html

```diff
@@ -40,12 +40,12 @@
                 </div>
             </div>
 
-            <div class="mt-5" *ngIf="isLoading">
-                <giddh-page-loader></giddh-page-loader>
+            <div class="position-center position-absolute" *ngIf="isLoading">
+                <giddh-page-loader class="unset-loader-margin"></giddh-page-loader>
```

Comment:
add this class declaration in global style

Reply:
Done

Reply:
Already exists.